app/src/ui/branches/push-branch-commits.tsx

```diff
@@ -71,12 +71,19 @@ export class PushBranchCommits extends React.Component<
   IPushBranchCommitsProps,
   IPushBranchCommitsState
 > {
+  private dialogButtonRef: HTMLButtonElement | null = null
   public constructor(props: IPushBranchCommitsProps) {
     super(props)
 
     this.state = { isPushingOrPublishing: false }
   }
 
+  public componentDidMount() {
+    if (this.dialogButtonRef) {
```

Comment:
Can we add a comment here explaining why we're doing this? something short is fine :)